modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -667,7 +667,7 @@ This parameter is also ignored unless a positive value is specified for either `
 
 For an overview of change history, see xref:learn:data/change-history.adoc[Change History].
 
-[#example-flushenabled-create]
+[#example-retention-collection-flushenabled-create]
```

Comment:
Need to remove `flushenabled-` bit from each of these anchors.  I think in each case the rest of the anchor is correct though.